.github/workflows/atlantis-image.yml

```diff
@@ -24,21 +24,21 @@ jobs:
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
-    # Publish latest image to container registry
-    - name: build atlantis:latest image
+    # Publish nightly image to container registry
+    - name: build atlantis:nightly image
```

Comment:
Why are you naming it nightly if it happens on every push to master? I like the `dev` idea a bit better

Reply:
ok, let's use dev then.